sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -1,75 +1,99 @@
 <template>
-  <v-row class="m-2">
-    <v-col class="category-header-medium color-blue">{{ header }}</v-col>
-  </v-row>
-  <div class="mx-5 py-4">
-    <content-group>
-      <DataTable
-        :autoLayout="true"
-        :value="applications"
-        class="p-m-4"
-        :paginator="true"
-        :rows="10"
+  <body-header
+    :title="header"
+    :subTitle="subTitle"
+    :recordsCount="disbursements.results?.length"
+    class="m-1"
+  >
+    <template #actions>
+      <InputText
+        type="text"
+        placeholder="Search name"
+        v-model="searchCriteria"
+        @keyup.enter="searchCOE"
+      />
+      <v-btn class="ml-2 primary-btn-background" @click="searchCOE"
+        ><font-awesome-icon :icon="['fas', 'search']" class="mr-2"
+      /></v-btn>
+    </template>
+  </body-header>
+  <content-group>
+    <DataTable
+      :value="disbursements.results"
+      :lazy="true"
+      class="p-m-4"
+      :paginator="true"
+      :rows="pageLimit"
+      :rowsPerPageOptions="rowsPerPageOptions"
+      :totalRecords="disbursements.count"
+      @page="pageEvent"
+      @sort="sortEvent"
+    >
+      <template #empty>
+        <p class="text-center font-weight-bold">No records found.</p>
+      </template>
+      <Column field="fullName" header="Name" sortable="true">
+        <template #body="slotProps">
+          <span>{{ slotProps.data.fullName }}</span>
+        </template>
+      </Column>
+      <Column field="studyStartPeriod" header="Study Period">
+        <template #body="slotProps">
+          <span>
+            {{ dateString(slotProps.data.studyStartPeriod) }} -
+            {{ dateString(slotProps.data.studyEndPeriod) }}
+          </span>
+        </template></Column
+      >
+      <Column field="applicationNumber" header="Application #"></Column>
+      <Column field="disbursementDate" header="Disbursement Date">
+        <template #body="slotProps">
+          <span>
+            {{ dateOnlyLongString(slotProps.data.disbursementDate) }}
+          </span>
+        </template></Column
       >
-        <template #empty>
-          <p class="text-center font-weight-bold">No records found.</p>
+      <Column field="coeStatus" header="Status" sortable="true">
+        <template #body="slotProps">
+          <COEStatusBadge :status="slotProps.data.coeStatus" />
+        </template>
+      </Column>
+      <Column field="applicationId" header="">
+        <template #body="slotProps">
+          <v-btn
+            :color="COLOR_BLUE"
```

Comment:
Are we be setting manually the color to these buttons?
Is it not supposed to be just a primary button?